webpack/assets/javascripts/jquery.ui.custom_spinners.js

```diff
@@ -0,0 +1,114 @@
+const megabyte = 1024 * 1024;
+const gigabyte = 1024 * megabyte;
+
+$(function () {
+  $.widget("ui.limited_spinner", $.ui.spinner, {
+    options: {
+      softMaximum: 0,
+      errorTarget: null
+    },
+    _validate: function() {
+      if (this.options['softMaximum'] != 0) {
+        if (this.value() > this.options['softMaximum']) {
+          this.options["errorTarget"].show();
+        } else {
+          this.options["errorTarget"].hide();
+        }
+      }
+    },
+    _spin: function (step, event) {
+      let result = this._super(step, event);
```

Comment:
Any reason to save this in a variable instead of just `return this._super(step, event);` after running the validation?